frontend/src/app/teams/components/layout/teams-layout.component.html

```diff
@@ -13,5 +57,11 @@
 </ng-container>
 
 <ng-template #teamErrorLayout>
-	<div class="teams__erorr">{{ errorMessage }}</div>
+	<div class="teams__error">
+		<h1>404</h1>
+		<div class="teams__error-content">
+			<h2>Oops.. <br> We couldn't find a team you are looking for.</h2>
+			<a href="/hub/overview" class="button">Back to the homepage</a>
```

Comment:
href is not the best practice for angular applications. use router link instead (read documentation)